### PR TITLE
Fix MANIFEST after #193

### PR DESCRIPTION
**Pull request by mattias-p**
Makefile.PL was converted to Module::Install so those files need to be
included.

Fixes a problem introduced in #193.

### CODE REVIEW

File: MANIFEST [hunk 1]
```diff
@@ -2,10 +2,10 @@
 Changes
 docker/Dockerfile
 docker/zonemaster_startup.sh
+docs/FAQ/gui-faq-da.md
 docs/FAQ/gui-faq-en.md
 docs/FAQ/gui-faq-fr.md
 docs/FAQ/gui-faq-sv.md
-docs/FAQ/gui-faq-da.md
```
Comment (matsduf) [inline MANIFEST]: You must not remove this!

File: MANIFEST [hunk 2]
```diff
@@ -95,11 +104,11 @@ zm_app/public/javascripts/dnscheck.js
 zm_app/public/javascripts/japp.js
 zm_app/public/javascripts/jquery-2.1.1.min.js
 zm_app/public/javascripts/jquery.js
+zm_app/public/lang/da.json
 zm_app/public/lang/en.json
 zm_app/public/lang/fr.json
 zm_app/public/lang/ru.json
 zm_app/public/lang/sv.json
-zm_app/public/lang/da.json
```
Comment (matsduf) [inline MANIFEST]: You must not remove this!

File: MANIFEST (hunk 1 above)
Comment (mattias-p) [reply MANIFEST]: I didn't. See line 5 after the update. When you run `make manifest` you get a lexicographically ordered MANIFEST file.

File: MANIFEST (hunk 2 above)
Comment (mattias-p) [reply MANIFEST]: See response above.

**Pull request merged**